test/verify/check-apps

```diff
@@ -95,6 +95,8 @@ class TestApps(packagelib.PackageCase):
         self.write_file("/etc/cockpit/apps.override.json",
                         '{ "config": { "appstream_data_packages": [ "appstream-data-test" ] } }')
 
+        self.createPackage("appstream", "1.0", "1", install=True)
```

Comment:
Yea, I am using Rocky, I dont have appstream nor appstream-data installed by default. That is what I was trying to solve, maybe there is some differences?

Reply:
Do you still see any value in these changes that way? Otherwise we can close it

Reply:
Ah, that explains it! Can you share your /etc/os-release, especially `ID` and `ID_LIKE`? We need to add Rocky and/or CentOS to https://github.com/cockpit-project/cockpit/blob/main/pkg/apps/manifest.json#L20 . I also think we shold check not just .ID, but also .ID_LIKE, which would hopefully take care of Centos, Rocky, and the like.

Reply:
> Do you still see any value in these changes that way?

Yes, I do. That alert is quite helpful to know what's going on. I am just pondering whether the page should even auto-refresh when there is no data, but that may be a little too aggressive.

Reply:
```
cat  /etc/os-release

NAME="Rocky Linux"
VERSION="9.2 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.2"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.2 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.2"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.2"
```

Reply:
CentOS 9 stream has
```
ID="centos"
ID_LIKE="rhel fedora"
```

and RHEL 9 is 
```
ID="rhel"
ID_LIKE="fedora"
```

Also, Ubuntu has 

```
ID=ubuntu
ID_LIKE=debian
```

I'll send a PR to clean this up and generalize it. this will already fix that aspect for you, I'll ping you in the PR.may

Reply:
Fixed in #19283

Reply:
Okay, so just to summarize, when I have time tonight to play with it:

My goal here is to keep the alert if `appstream-data` is missing and use the manifest reference for it, right? And then update integration tests accordingly.

And of course, drop the install procedure since it is already in place.

Reply:
Correct. Your check for "are the metapackages installed" looked quite fine.

Reply:
Reworked, I did move some stuff around so feel free to do another pass